test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -27,19 +29,24 @@ public void ValidateBasicRequest(InProcessServer server, string requestPath, Req
             var httpClientHandler = new HttpClientHandler();
             httpClientHandler.UseDefaultCredentials = true;
 
-            var httpClient = new HttpClient(httpClientHandler, true);
-            var task = httpClient.GetAsync(server.BaseHost + requestPath);
-            task.Wait(TestTimeoutMs);
+            Task<HttpResponseMessage> task;
+            using (var httpClient = new HttpClient(httpClientHandler, true))
```

Comment:
Thank-you for fixing stuff like this, clearly I need to add an FxCop pass to my TODO list for this SDK.